src/main/java/org/mkbservices/models/UserModel.java

```diff
@@ -0,0 +1,97 @@
+package org.mkbservices.models;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+//this is the model class
+public class UserModel {
+    // these are the fields
+    private String firstName;
+    private String lastName;
+    private String DOB;
+    private String username;
+    private Integer age;
+    public static List<UserModel> users = new ArrayList<>();
+
```

Comment:
- I am wondering why you added this users list
- DOB is less meaningful as a field name, have clear variable names that follow the naming convention.